content/en/design/Interface/typography.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Typography"
+linkTitle: "Typography"
+weight: 3
+description: >
+  Typography is the design of type in ways that provides innate hierarchy to UI
+---
+## Typography
+
+The default app font is <a href="https://www.google.com/get/noto/">Noto Sans</a>. It is free, ocodepen source, supports 800 languages and was specifically designed for good web legibility. It is bundled with the app so that all users see the same font regardless of their particular device, language, browser, etc. This ensures a consistent experience for all users.
```

Comment:
```suggestion
The default app font is <a href="https://www.google.com/get/noto/">Noto Sans</a>. It is free, codepen source, supports 800 languages and was specifically designed for good web legibility. It is bundled with the app so that all users see the same font regardless of their particular device, language, browser, etc. This ensures a consistent experience for all users.
```